ui/apps/platform/src/services/DelegatedRegistryConfigService.ts

```diff
@@ -0,0 +1,16 @@
+import axios from './instance';
+
+import { DelegatedRegistryConfig } from '../types/dedicatedRegistryConfig.proto';
```

Comment:
Super that this file name corresponds to backend proto:
https://github.com/stackrox/stackrox/blob/master/proto/api/v1/delegated_registry_config_service.proto

Because backend pattern for new API since 2022 has been payload and response types encapsulated with service endpoints, new frontend service files also include those types to lower the barriers:
* for reviewers like me to double-check the TypeScript
* for backend teammates to expedite progress by adding properties to both files at the same time